### DISCUSSION TITLE
Mobile app sharing and web clipping

### DISCUSSION THREAD

**Discussion by lx6g** [standardnotes/forum#2697] [Suggestions]
Seen few of other issues in the tracker, but mixed along other questions so asking here:

1. Do you not really offer a "Share with Standard Notes" feature for Android / iOS versions? Like saving a tweet, link, text from almost any other app to SN? That is hugely useful and probably a blocker for many, me - definitely.

2. Web clipper. At least as a plug in for Firefox / Chrome / Safari on desktops and ideally something similar on mobiles as well. This is also very important - to create your own catalogue of articles and web pages, safely and privately saved to your notebook.

**Comment by aymza** [comment]
Same: unfortunately unusable since it does not appear in the Android "share" menu. Hope you can work on that soon !

**Comment by JaspalSuri** [comment]
Thanks for your feedback, I've passed it along to the team! Believe me when I say that we (at least) want the first feature as much as you do!

**Comment by effieeee** [comment]
Hi there, regarding the web clipper, another user actually managed to develop an extension for that on Firefox and Chrome! It's worth a try, and they're still actively working on it! You'll find it in the repository below, including the installation instructions!

https://github.com/johnjones4/Standard-Notes-Clipper

**Comment by tracyboehrer** [comment]
+1

Trying to dump Google Keep, and dislike Pocket.  But a large number of my notes are links to sites or images when I'm on mobile.  Lack of share is a huge usability problem on mobile.  I could live with having to copy/paste a URL (though that is more steps than I want to take on mobile).  But in the case of a photo...  ugh... just auto add to FileSafe and save the note with the attachment.

**Comment by JaspalSuri** [comment]
> But in the case of a photo... ugh... just auto add to FileSafe and save the note with the attachment.

Thanks for the feedback! I've forwarded your request to the team. :)

**Comment by trx314** [comment]
Sharing via Android "share" menu is a must!!! trying to get rid of Keep, but this one is simply to hard to avoid, I spend my time saving links and tweets for later analysis, and share>>keep is the most natural way to do it.

Is there an ETA for this feature?

**Comment by JaspalSuri** [comment]
Currently no ETA, but the dev team is currently working on rewriting all of the apps and the codebase that they build upon. Once that's complete, they can start working on adding new features like sharing content to Standard Notes on mobile platforms.

**Comment by FetchFast** [comment]
My goodness. It's been a long time. Are there any coders here? I'm not a programmer but it seems easy...
Though I'm sure we've heard that one before!

**Comment by JaspalSuri** [comment]
IIRC, one of the challenges of sharing content to the app is that it's difficult to decrypt all of the user's locally stored data when sharing content from outside of the app in order to create a new note. Implementing this feature is still a part of our roadmap. 

As for the web clipper, we currently do not plan to work on (or improve upon) one, given our limited development bandwidth and current focus on our core products.

**Comment by zosoguidi** [comment]
Any news on (re)introducing sharing into to app? still on the roadmap? this feature would have a huge impact on user activity, engagement, usage, growth......... for many it's a must have.

**Comment by Alok-S-Sharma** [comment]
Hi there,

+1 to Share with Standard Notes on android.

I desperately need this feature because I have to save lot of useful information in Standard Notes.

Presently, I have to copy & paste contents in Standard Notes. It adds extra steps and very inconvenient.

**Comment by zosoguidi** [comment]
Hey hey! Sharing feature added since version 3.166 ! Yes!
